analysis_options.yaml

```diff
@@ -11,3 +11,11 @@ linter:
     non_constant_identifier_names: false
 
     constant_identifier_names: false
+
+    no_leading_underscores_for_local_identifiers: false
```

Comment:
I don't think we need these changes in analysis_options, do we ?

Reply:
These are the new linter rule introduced in the latest version of the `flutter_lints`, and that requires changes of around 500 lines. So, I have disabled this for the current PR. Otherwise, this will fail the CI.

Reply:
And those changes are related to this issue #149

Reply:
Ok, please also create another Issue for fixing these rules